### ISSUE TITLE
"An error occured parsing the response for image page"

### ISSUE THREAD

**Issue by riesza** [dnsev-h/eze#23]
![atocet 1](https://user-images.githubusercontent.com/1399263/44149457-766df798-a09c-11e8-9a09-f4e39baefea9.png)
Getting this quite often, despite being able to load the images manually just fine. It just kind of locks itself in that state even if I retry it until I try again a couple hours later. Sometimes opening some of the images it gets stuck on manually helps it get past it and it either locks up at some point later in the gallery or it actually finishes it.

FF 62.0b17
ViolentMonkey v2.9.5
Script v1.0.9

**Comment by ginx2666** [comment]
Don't count on reply, it's a dead project.

**Comment by dnsev-h** [comment]
Do you have any example URLs it consistently fails on? Or is it hard to reproduce?

There are some deeply ingrained issues with how this script downloads galleries that can lead to this problem, but in the past this has typically only happened when downloading very large galleries.

<br />

> Don't count on reply, it's a dead project.

rude

**Comment by ginx2666** [comment]
Yes, it's rude how I have to make a scathing comment to get a reply to an ongoing issue report that's months old.

Anyway, _any_ new gallery (I'm talking very new, as in on e-h, but not yet visible in ex) has a chance to display that either immediately, or at pulling an image. After 8-24 hours it pulls images correctly. If the script had a logging function, a log could be posted.

**Comment by beriah** [comment]
Reproducing it is difficult because it seems random. But if a gallery fails once it fails consistently for a while for me. It seems that waiting a couple days does help most of the time but not always, as I've had several days old galleries still fail with that error. Most of the time it errors out immediately like ginx said.

**Comment by dnsev-h** [comment]
I can actually guarantee that the issue is older than just "months". Probably been there from the start.

Part of the problem comes from the way the script pre-caches the download URLs all at the start. This is not a safe thing to do since these links expire. This is what causes this issue to occur on large galleries.

The other part of the problem is the H@H servers sometimes not responding. This leads to timeout errors which can be annoying to deal with.

Another issue I've noticed with Firefox is that as it updated to using web extensions, most of the userscript engines don't handle this script well anymore when trying to download images. For some reason, this can cause insane memory usage and application lag.

This all leads to making error reproduction rather annoying. I.e. I cannot reliably get the script to fail on my end with some basic tests.

**Comment by fullmetal1** [comment]
Just wanted to note that this has started happening with every download recently. Here's an extra big gallery that's been causing issues: [The Art of The Wind Rises](https://e-hentai.org/g/2654145/626e28ec82/)
I'm on FF 116.0.2 on Linux with Violentmonkey 2.15.0 and Eze 1.0.9.3, if that helps.
